next permutation in c++

```diff
@@ -0,0 +1,69 @@
+#include <bits/stdc++.h>
+using namespace std;
+
+//to print a vector
+void print(vector<int> a,int n){ 
+	for(int i=0;i<n;i++)
+		cout<<a[i]<<" ";
+	cout<<endl;
+}
+
+void nextPermutation(vector<int> a,int n){
```

Comment:
Comments for the code below are largely similar to my comments above, so I'll not repeat them :)